### PR TITLE
Create Getting_Started_CLI.md

### PR DESCRIPTION
**Pull request by DanMayhew**
Plan to add a new folder: Documentation and add to the folder markdown files named: Getting_Started_CLI, Getting_Started_Desktop_App, and Getting_Started_Docker.  This is the first file, but need to learn how to add color to words for the code.

# Description
Describe the goals of the PR.
# Related PRs
List related PRs with links.
# Steps to Test or Reproduce
Outline the steps to test or reproduce PR.
# Impacted Areas in Application
List areas of Cauldron this PR will affect.

### CODE REVIEW

**Comment by DanMayhew** [comment]
Added system requirements to Getting Started CLI and broked out all the getting started documentation into separate files.  Please check the links to install the desktop and Linux versions are correct.

File: Getting_Started_CLI.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+  
+Python 2 is nearing its end of life and lacks the necessary functionality to be supported.
+  
+### 64 Bit OS
+The desktop application is available for OSX, Windows and multiple Linux distributions, but they must be the 64 bit versions.
+
+## Installation
+The latest release of Cauldron is available from both PyPi:
+
+<pre>$ pip install cauldron-notebook</pre>
```
[the remark this answers is not in the document]
Comment (DanMayhew) [reply Getting_Started_CLI.md]: Changed all pre tags to ```bash.

File: Getting_Started_Desktop_App.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+The package can be installed either using conda for Anaconda users:
+  
+<pre>$ conda install -c sernst cauldron$</pre>
+  
+or from Python's official package index (PyPI) using pip:  
+  
+<pre>$ pip install cauldron-notebook$</pre>
+
+
+## 2 Desktop Application
```
[the remark this answers is not in the document]
Comment (DanMayhew) [reply Getting_Started_Desktop_App.md]: Done. Added link to unnotebook.com

File: Getting_Started_Linux.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# Getting Started Linux(*.deb)Installation
```
[the remark this answers is not in the document]
Comment (DanMayhew) [reply Getting_Started_Linux.md]: Deleted file and added link to unnotebook.com for linux.

[the remark this answers is not in the document]
Comment (sernst) [reply Getting_Started_Linux.md]: 👍

File: DOCUMENTATION/Getting_Started_CLI.md [hunk 4]
```diff
@@ -0,0 +1,310 @@
+# Getting Started with the CLI
+
+Cauldron can be used as either through its Command Line Interface (CLI) or with the Cauldron desktop application. This document explains using Cauldron through its CLI. To install the Cauldron desktop app you can instructions here: http://www.unnotebook.com/install.
```
Comment (sernst) [inline DOCUMENTATION/Getting_Started_CLI.md]: Missing word here: *To install the Cauldron desktop app you can **?** instructions ...*

Comment (DanMayhew) [reply DOCUMENTATION/Getting_Started_CLI.md]: Corrected the sentence to: <em>To install the Cauldron desktop app you may view the instructions here: </em>

**Comment by DanMayhew** [comment]
I did delete the README.rst file.  Can you reject that commit, or do I need to revert that file?  Not sure how to revert.

**Comment by DanMayhew** [comment]
Figured out how to do a revert and added back the README.rst file.

**Pull request merged**